inc/ModelBasedPrognoser.h

```diff
@@ -5,15 +5,16 @@
  *
  *   @brief     Model-based Prognoser Class
  *
- *   General model-based prognoser class. It gets created for a specified model, observer, and
- * predictor.
+ *   General model-based prognoser class. It gets created for a specified model, observer,
+ * predictor, and load-estimator.
  *
  *   @author    Matthew Daigle
+ *	 @author 	Christopher Teubert
  *   @version   1.1.0
  *
- *   @pre       Prognostic Configuration File and Prognoster Configuration Files
+ *   @pre       Prognoster Configuration Files or all of the following: model, observer, predictor, load estimator
  *
- *      Contact: Matthew Daigle (matthew.j.daigle@nasa.gov)
+ *      Contact: Chris Teubert (christopher.a.teubert@nasa.gov)
```

Comment:
I prefer to change these to list all authors. Contact for someone outside of the group is really "open an issue on Github", and the appropriate author is usually obvious within the group.